lib/knapsack_pro/report.rb

```diff
@@ -30,12 +36,19 @@ def self.save_subset_queue_to_file
       end
     end
 
-    def self.save_node_queue_to_api
+    def self.get_test_files
```

Comment:
I worry that the `get_test_files` method name overlaps with `get_tests` and it might be confusing why we have 2 methods :/.

The following method is strictly related to queue mode.

```suggestion
    def self.test_files_recorded_in_queue_mode
```

Reply:
I've inlined it.

The fact that we need to add suffixes to methods in that file is a smell for an incorrect abstraction. But I won't look into it.